src/main/java/com/grasshouse/dorandoran/post/service/dto/PostResponse.java

```diff
@@ -0,0 +1,32 @@
+package com.grasshouse.dorandoran.post.service.dto;
+
+import com.grasshouse.dorandoran.member.service.dto.MemberResponse;
+import com.grasshouse.dorandoran.post.domain.Address;
+import com.grasshouse.dorandoran.post.domain.Location;
+import com.grasshouse.dorandoran.post.domain.Post;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class PostResponse {
+
+    private Long id;
+    private MemberResponse memberResponse;
+    private String content;
+    private Location location;
+    private Address address;
+
+    public static PostResponse of(Post post) {
+        return PostResponse.builder().id(post.getId())
+            .memberResponse(MemberResponse.of(post.getAuthor()))
+            .content(post.getContent())
+            .location(post.getLocation())
+            .address(post.getAddress())
+            .build();
```

Comment:
```suggestion
        return PostResponse.builder()
            .id(post.getId())
            .memberResponse(MemberResponse.of(post.getAuthor()))
            .content(post.getContent())
            .location(post.getLocation())
            .address(post.getAddress())
            .build();
```

`builder()` 다음에는 한줄 내리는 게 어떨까요?

Reply:
동의합니다! 반영하겠습니다.